### PR TITLE
Replace the subctl info to show networks cmd

### PR DESCRIPTION
**Pull request by maayanf24**
Signed-off-by: Maayan Friedman <maafried@redhat.com>

### CODE REVIEW

**Comment by maayanf24** [comment]
Related Issue #190

**Comment by tpantelis** [comment]
This is a draft - is it ready for review now?

**Comment by maayanf24** [comment]
> This is a draft - is it ready for review now?

I will add to this PR the other show cmds and then make it ready for review

**Comment by mangelajo** [comment]
LGTM!

It would be nice to add output of the commands (that deserves another PR and not only specific to what you are adding, but the document in general).

File: src/content/deployment/subctl.en.md [hunk 1]
```diff
@@ -90,14 +90,46 @@ broker-info file.
...
+
+The `show networks` command inspects the cluster and reports information related to Submariner, like the
 detected network plugin, and the detected Cluster and Service CIDRs.
 
-#### info flags
+#### show versions
+
+`subctl show versions [flags]`
+
+The `show versions` command shows the version and repository of the submariner components in the cluster.
```
Comment (tpantelis) [inline src/content/deployment/subctl.en.md:106]:
```suggestion
Shows the version and image repository of each Submariner component in the cluster.
```

File: src/content/deployment/subctl.en.md [hunk 2]
```diff
@@ -90,14 +90,46 @@ broker-info file.
...
+### show
 
-`subctl info [flags]`
+#### show networks
 
-The `info` command inspects the cluster and reports information related to Submariner, like the
+`subctl show networks [flags]`
+
+The `show networks` command inspects the cluster and reports information related to Submariner, like the
 detected network plugin, and the detected Cluster and Service CIDRs.
```
Comment (tpantelis) [inline src/content/deployment/subctl.en.md:100]:
```suggestion
Inspects the cluster and reports information about the detected network plugin and detected Cluster and Service CIDRs.
```

File: src/content/deployment/subctl.en.md [hunk 3]
```diff
@@ -90,14 +90,46 @@ broker-info file.
...
+
+`subctl show versions [flags]`
+
+The `show versions` command shows the version and repository of the submariner components in the cluster.
+
+#### show gateways
+
+`subctl show gateways [flags]`
+
+The `show gateways` command shows summary information about the submariner gateways in the cluster.
```
Comment (tpantelis) [inline src/content/deployment/subctl.en.md:112]:
```suggestion
Shows summary information about the Submariner gateways in the cluster.
```

File: src/content/deployment/subctl.en.md [hunk 4]
```diff
@@ -90,14 +90,46 @@ broker-info file.
...
+ 
+#### show connections
+
+`subctl show connections [flags]`
+
+The `show connections` command shows information about submariner endpoint connections with other cluster
```
Comment (tpantelis) [inline src/content/deployment/subctl.en.md:118]:
```suggestion
Shows information about the Submariner endpoint connections with other clusters.
```

File: src/content/deployment/subctl.en.md [hunk 5]
```diff
@@ -90,14 +90,46 @@ broker-info file.
...
+
+#### show endpoints
+
+`subctl show endpoints [flags]`
+
+The `show endpoints` command shows information about submariner endpoints in the cluster.
```
Comment (tpantelis) [inline src/content/deployment/subctl.en.md:124]:
```suggestion
Shows information about the Submariner endpoints in the cluster.
```

File: src/content/deployment/subctl.en.md [hunk 6]
```diff
@@ -90,14 +90,46 @@ broker-info file.
...
+
+#### show all
+
+`subctl show all [flags]`
+
+The `show all` command shows the information from all other show commands.
```
Comment (tpantelis) [inline src/content/deployment/subctl.en.md:130]:
```suggestion
Shows the aggregated information from all the other show commands.
```

**Pull request merged**